CHANGELOG.rst

```diff
@@ -6,10 +6,15 @@ dcicutils
 Change Log
 ----------
 
+1.16.1
```

Comment:
Feels like this should be a major version bump. I guess if you want to declare there are no users of this yet, we could not do that. But then I would put it in doc somewhere that the modules you're playing with are experimental and not yet something for which we're bumping major versions on incompatible changes.